idalib/src/idb.rs

```diff
@@ -213,6 +214,36 @@ impl IDB {
         }
     }
 
+    pub fn get_cmt(&self, ea: Address, rptble: bool) -> String {
+        unsafe { idalib_get_cmt(ea.into(), rptble) }
+    }
+
+    pub fn set_cmt(
```

Comment:
Similar to `append_cmt`, I'd create a second variation of this API with a default value for non-repeatable comments.